probe-rs/src/bin/probe-rs/cmd/dap_server/server/core_data.rs

```diff
@@ -35,13 +35,13 @@ pub struct CoreData {
     ///   To ensure the [`CoreHandle::poll_core()`] behaves correctly, it will set the `last_known_status` to [`CoreStatus::Running`],
     ///   and execute the request normally, with the expectation that the core will be halted, and that 1. above will detect this new status.
     ///   These 'implicit' updates of `last_known_status` will not(and should not) result in a notification to the client.
-    pub(crate) last_known_status: CoreStatus,
```

Comment:
Why do those need to be pub?

Reply:
This is a binary, so there is not really a difference between `pub` and `pub(crate)`. I just felt that it looks cleaner :)
Can undo it if you think it's not a good idea.

Reply:
No, that's a valid point, I thought that many times :)